packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -0,0 +1,223 @@
+// This notifier is used to monitor and compare Optimistic Oracle contract proposals against the Polymarket API endpoint.
+
+const { Datastore } = require("@google-cloud/datastore");
+const datastore = new Datastore();
+const abi = require("./abi/abi");
+const Web3 = require("web3");
+const ethers = require("ethers");
+const uma = require("@uma/sdk");
+const { getAddress } = require("@uma/contracts-node");
+
+const web3 = new Web3(process.env.CUSTOM_NODE_URL);
+const provider = new ethers.providers.WebSocketProvider(process.env.CUSTOM_NODE_URL);
+
+const polymarketContract = "0xCB1822859cEF82Cd2Eb4E6276C7916e692995130";
+const earlyExpiryResponse = "-57896044618658097711785492504343953926634992332820282019728792003956564819968";
```

Comment:
this is in the consts file already and should not be defined like this. see https://github.com/UMAprotocol/protocol/blob/ceb5a468d6cfb9cf1996fb6ad058ac36d5f7db65/packages/common/src/Constants.ts#L46